templates/eth1tx.html

```diff
@@ -1,4 +1,19 @@
 {{ define "js" }}
+  <script>
+    function togglePriceContent(currentPrice, historicPrice) {
+      var content = document.getElementById("valuePriceButton")
+      var curValTitle = "Current Value"
+      var estValTitle = "Estimated Value on Day of Txn"
+
+      if (content.getAttribute("data-original-title") === estValTitle) {
```

Comment:
Super minor detail I noticed, tooltip won't update until you move the mouse from the element and back to it. Not sure if we can fix this behavior and change the tooltip on click.

Reply:
Fixed.